utility/WiFiStream.h

```diff
@@ -135,6 +135,14 @@ class WiFiStream : public Stream
   {
     return _client.status();
   }
+#elif ARDUINO_ARCH_ESP32
+  /**
+   * check if TCP connection is established
+   */
+  inline uint8_t connected()
```

Comment:
Why are you creating an ESP32 specific API?

Reply:
The ESP32 SDK does not provide the same functionality as the ESP8266 SDK. For some use cases it is still relevant to know if the stream is connected, as this is not the same as checking the WiFi status. One alternative would be to mimic the ESP8266 implementation and create a status value from the bool provided by the ESP32 SDK.

Reply:
I think that's the way to go, especially since there is precedent.

Reply:
> I think that's the way to go.

You refer to the alternative I mentioned (to use the previous interface)?

Reply:
Done, see latest commit. Also added an enum for the possible return values of method status() as the ESP32 SDK does not define constants like the ESP8266 SDK does.